File: frameworks/apollo-server/src/entities/Post.ts
import { Entity, PrimaryColumn, Column } from 'typeorm';

@Entity({ schema: 'benchmark', name: 'tb_post' })
export class Post {
  @PrimaryColumn('uuid')
  id!: string;

  @Column('text')
  title!: string;

  @Column('text', { nullable: true })
  content!: string | null;

  @Column('uuid')
  author_id!: string;

  @Column('timestamp')
  updated_at!: Date;
}

File: frameworks/spring-boot-orm/src/main/java/com/fraiseql/entities/Post.java
package com.fraiseql.entities;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;
import lombok.Builder;

import javax.persistence.*;
import java.time.LocalDateTime;
import org.springframework.data.annotation.LastModifiedDate;
import org.springframework.data.jpa.domain.support.AuditingEntityListener;

@Entity
@Table(name = "tb_post", schema = "benchmark")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@EntityListeners(AuditingEntityListener.class)
public class Post {

    @Id
    @Column(name = "pk_post")
    private Integer pkPost;

    @Column(name = "id")
    private String id;

    @Column(name = "fk_author")
    private Integer fkAuthor;

    @Column(name = "title")
    private String title;

    @Column(name = "content")
    private String content;

    @Column(name = "published")
    private Boolean published;

    @Column(name = "created_at")
    private LocalDateTime createdAt;

    @LastModifiedDate
    @Column(name = "updated_at")
    private LocalDateTime updatedAt;
}